src/deploy/index.js

```diff
@@ -105,6 +105,7 @@ var deploy = function(targetNames, options) {
         _.each(context.hosting.deploys, function(deploy) {
           logger.info(clc.bold("Hosting URL:"), utils.addSubdomain(api.hostingOrigin, deploy.site));
         });
+        return { hosting: context.hosting.deploys.map((deploy) => deploy.version) };
```

Comment:
I think we should do

```suggestion
        const versionNames = context.hosting.deploys.map((deploy) => deploy.version);
        return { hosting: (versionNames.length === 1 ? versionNames[0] : versionNames) };
```

this keeps the API identical(ish) to the previous behavior for single-site, and provides an array of version names when multiple sites are deployed.